src/cards.scss

```diff
@@ -0,0 +1,139 @@
+$secondary: #616070;
+$btn-white-color: #fff;
+$btn-turquoise-color: #00acdc;
+$main-acent: #060b35;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  width: 1200px;
```

Comment:
The width should not be set on the `body` element directly. It can lead to unexpected layout issues. Consider using a container element inside the `body` to control the width of the content.